visidata/basesheet.py

```diff
@@ -24,6 +25,15 @@ def keys(self):
     def clear(self):
         self.locals.clear()
 
+    def __len__(self):
+        return len(set().union(self.objs, self.locals))
```

Comment:
I'm thinking maybe `keys()` above should return the `list(union(...))` mentioned in the iter comment.  Then this can return `len(self.keys())`.